### DISCUSSION TITLE
Blurry Battery and Network Taskbar Icons

### DISCUSSION THREAD

**Discussion by Ik549** [valinet/ExplorerPatcher#2231] [Q&A] [answered]
![Screenshot 2023-10-02 143505](https://github.com/valinet/ExplorerPatcher/assets/76847411/83d9fcc5-c7c5-4056-a840-c0c621d0fc59)
Hi, The taskbar battery and network icons are a bit blurry usually restarting file explorer in ep fixed it but not anymore i dont know if this has been addressed earlier Ive checked, but I'd appreciate some help

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by Ik549** [reply]
Version is 22H2
OS Build 19045.3516
Microsoft Windows


Explorer Patcher Version 22621.2361.58.1
![Screenshot 2023-10-03 161818](https://github.com/valinet/ExplorerPatcher/assets/76847411/1f6ac215-0ef1-4f33-880d-6e2028b9b3c0)
![Screenshot 2023-10-03 161851](https://github.com/valinet/ExplorerPatcher/assets/76847411/baf3891e-2df9-4d8b-b55f-2686e63aab82)
![Screenshot 2023-10-03 161912](https://github.com/valinet/ExplorerPatcher/assets/76847411/0894dc3f-9563-4a79-842e-2991c5edc503)
![Screenshot 2023-10-03 161927](https://github.com/valinet/ExplorerPatcher/assets/76847411/43d9f166-f526-4678-bb9a-c39ff78a53bc)

These are all the settings I believe I set I haven't touched any of the other categories

**Comment by Poopooracoocoo** [comment]
I noticed that your battery icon is coloured. How did you modify it? And what resolution and display scale do you have?

**Reply by Ik549** [reply]
its modified with the battery mode app https://github.com/tarcode-apps/BatteryMode ,  my scale is at 100%recommended and resolution is 1366x768recommended

**Reply by pyrates999** [reply]
I don't think this is caused by EP, but by the batteryMode application.

**Reply by Poopooracoocoo** [reply]
But they also noticed this with the network icon

**Reply by Ik549** [reply]
This happens regardless, whether I'm running Battery mode or not, I also use 7+ Taskbar Tweaker(https://ramensoftware.com/7-taskbar-tweaker) and this happens when the app isn't running either. My best guess is software interaction because of the number of taskbar apps I'm using, the only way I've been able to fix it in the past was by restarting file explorer from ExplorerPatcher Properties which doesn't work anymore(the fix doesn't work anymore).

 Thanks for replying as well @Poopooracoocoo  @pyrates999

**Reply by pyrates999** [reply]
I would uninstall the batteryMode application, reboot, and see if it is still happening.

**Comment by K4ldun** [comment]
~23H2~
~22631.2506~
~22621.2428.59.1~

~same issue, after restart icons are blurry until explorer is restarted.~

~Upon startup:~
![image](https://github.com/valinet/ExplorerPatcher/assets/16339619/22ced0a1-5462-4a8a-9e9c-c9b935ae9161)

~After Explorer restart:~
![image](https://github.com/valinet/ExplorerPatcher/assets/16339619/a8a08009-e2fe-43f6-871f-8903169ac79f)

**Reply by K4ldun** [reply, marked as answer]
False alarm, the reason this was happening was that I had used a software called Winaero Tweaker before and changed system tray related settings there. Upon resettin them everything's back to normal.

**Reply by pyrates999** [reply]
Please mark this as the answer then.

**Reply by Poopooracoocoo** [reply]
That wasn't OP that commented so OP cannot mark that as an answer

**Reply by pyrates999** [reply]
ah right :)  Sorry :)

**Reply by Ik549** [reply]
could you share the settings you resent in winaero i also use it
